frameworks/Go/fiber/benchmark_config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "framework": "fiber",
+  "tests": [{
+    "default": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
+      "database": "None",
+      "framework": "None",
```

Comment:
`fiber` as the framework here would be helpful

Reply:
Done